ui/lib/core/addon/components/chevron.js

```diff
@@ -12,25 +18,17 @@
  * @param [isButton=false] {String} - if true, adjusts the CSS classes to push the icon closer to the right of a button.
  *
  */
-import Component from '@ember/component';
-import { computed } from '@ember/object';
-import { assert } from '@ember/debug';
 
-import layout from '../templates/components/chevron';
-
-const DIRECTIONS = ['right', 'left', 'up', 'down'];
+export default class Chevron extends Component {
+  tagName = '';
```

Comment:
<img width="1366" alt="Screenshot 2023-03-13 at 2 44 09 PM" src="https://user-images.githubusercontent.com/30884335/224842048-b607b1bb-bda3-45f7-b661-036c98f1ff56.png">
when I removed `tagName`, it caused some styling issues in the ui^ I'm not sure why there were issues when `tagName` was removed.  From my understanding tagName is not needed for Ember Octane https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/#component-templates__tag-name